aws/src/main/java/org/apache/iceberg/aws/glue/IcebergToGlueConverter.java

```diff
@@ -276,4 +298,11 @@ private static void addColumnWithDedupe(List<Column> columns, Set<String> dedupe
       dedupe.add(field.name());
     }
   }
+
+  private static Set<String> getAdditionalLocations(TableMetadata tableMetadata) {
```

Comment:
I think this method is simple enough that we can just put it in the main method, no need to have a separated helper